apps/redi-connect/src/components/organisms/ApplicationCard.tsx

```diff
@@ -42,10 +42,15 @@ const ApplicationCard = ({
     profile.userType === 'mentee' ? application.mentor : application.mentee
   const currentUserIsMentor = currentUser?.userType === 'mentor'
 
+  const getClassName =
```

Comment:
@katamatata, suggest rename to simply `className`. To me the `get` prefix implies this is a function.

Reply:
Further down (line 100) there's a similar expression to pick a className, but it's inline. Consider doing the same here.

Reply:
You'd prefer me to move `getClassName` expression to inline `className` like on line 100. Did I get it right?

Reply:
Precisely